presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -937,4 +938,17 @@ public HiveConfig setProjectionPushdownEnabled(boolean projectionPushdownEnabled
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    public boolean getGrantOption()
+    {
+        return grantOption;
+    }
+
+    @Config("hive.grant-option")
```

Comment:
IMO `hive.create-with-grant-option` would be more descriptive.

Reply:
I think this be should defined only when `hive.security=sql` is turned on. That means that we might need `SqlStandardAccessControlConfig` that is only loaded in `SqlStandardSecurityModule`.